spec_cleaner/rpmpreamble.py

```diff
@@ -488,7 +533,9 @@ def add(self, line):
                 self._add_line_to('define', line)
 
             # catch "modname" for use in pypi url rewriting
-            if (line.startswith('%define') or line.startswith('%global')) and line.find('modname') >= 0:
+            if (line.startswith('%define') or line.startswith('%global')) and line.find(
+                'modname'
+            ) >= 0:
```

Comment:
this was much more readable before the blacken).

Reply:
Fixed by a higher line length limit (100 chars).